src/index.js

```diff
@@ -23,14 +23,19 @@ class Nouislider extends React.Component {
   }
 
   shouldComponentUpdate(nextProps) {
-    const { start, disabled } = this.props;
-    return !isEqual(nextProps.start, start) || nextProps.disabled !== disabled;
+    const { start, disabled, range } = this.props;
+    return (
+      !isEqual(nextProps.start, start) ||
+      nextProps.disabled !== disabled ||
+      nextProps.range !== range
```

Comment:
**range** is object, so it will be true - and this is not what I actually want.
You can use ** isEqual** from utils, but you should also make some changes to support objects

Reply:
Thanks @mmarkelov, sorry for the mistake. I fixed it in the last commit. I used a simple function to compare objects. Maybe in the future it will be interesting to use a more robust method like _.isEqual from lodash.

Reply:
@alexmasnou I should think about it. I don't want to include this kind of dependencies and want to make library as simple as possible